src/modules/python.rs

```diff
@@ -93,14 +93,24 @@ fn get_pyenv_version(context: &Context) -> Option<String> {
 }
 
 fn get_python_version(context: &Context, config: &PythonConfig) -> Option<String> {
-    let version = config
+    config
         .python_binary
         .0
         .iter()
-        .find_map(|binary| context.exec_cmd(binary, &["--version"]))
-        .map(get_command_string_output)?;
-
-    parse_python_version(&version)
+        .find_map(|binary| {
+            let command = binary.0[0];
```

Comment:
```suggestion
            let command = binary.0.get(0)?;
```

Reply:
Please also update the config schema via `cargo run --features config-schema -- config-schema > .github/config-schema.json`.